sdk/keyvault/azure-security-keyvault-secrets/src/main/java/com/azure/security/keyvault/secrets/SecretAsyncClient.java

```diff
@@ -64,22 +49,11 @@
  * @see SecretClientBuilder
  * @see PagedFlux
  */
-@ServiceClient(builder = SecretClientBuilder.class, isAsync = true, serviceInterfaces = SecretService.class)
+@ServiceClient(builder = SecretClientBuilder.class, isAsync = true, serviceInterfaces = SecretClientImpl.SecretService.class)
```

Comment:
I think we should remove `serviceInterfaces` rather than pointing to impl types. (that's how anomaly detector seem to look like)